contracts/ContinuousFundraising.sol

```diff
@@ -114,6 +130,19 @@ contract ContinuousFundraising is ERC2771Context, Ownable2Step, Pausable, Reentr
         );
         require(_tokenPrice != 0, "_tokenPrice needs to be a non-zero amount");
         require(_maxAmountOfTokenToBeSold != 0, "_maxAmountOfTokenToBeSold needs to be larger than zero");
+        currencyReceiver = _currencyReceiver;
+        minAmountPerBuyer = _minAmountPerBuyer;
+        maxAmountPerBuyer = _maxAmountPerBuyer;
+        tokenPrice = _tokenPrice;
+        maxAmountOfTokenToBeSold = _maxAmountOfTokenToBeSold;
+        currency = _currency;
+        token = _token;
+        // Initialize owner by writing the private variable `_owner` of the Ownable contract using assembly.
+        // This will re-write the storage slot if the contract is created through the constructor, which is acceptable.
+        require(_owner != address(0), "owner can not be zero address");
+        assembly {
```

Comment:
Why is this necessary?

Reply:
If the contract is created through the constructor, openzeppelin's ownable does this for us:
`require(newOwner != address(0), "Ownable: new owner is the zero address");`
We should provide the same protection when creating a clone.